### PR TITLE
Request for addition to the list of baggers

### PR DESCRIPTION
**Pull request by rtouze**
Hi, I propose 3 BBL sessions in Brest.

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -15823,6 +15823,67 @@ var data = {
         "twitter": "cledanjou",
         "mail": "clement@danjou.io"
       }
+    },
+    {
+      "since": "2017-10-11",
+      "name": "Romain Touzé",
+      "bio": "Développeur fullstack",
+      "picture": "https://s3-eu-west-1.amazonaws.com/rtodev.tech/img/avatar_tourne.jpg",
+        "websites": [
```
Comment (dadoonet) [inline baggers.js]: nit: not aligned with `"picture"`

**Review by dadoonet — APPROVED** [review]
I left a small comment. If you can fix before we merge that'd perfect. Otherwise, we can merge it like this. No worries.

**Comment by rtouze** [comment]
Done!

**Comment by dadoonet** [comment]
Welcome !

**Comment by rtouze** [comment]
OK, it broke the website... I forgot the cities part in my profile, sorry. I'll open another PR.

**Comment by dadoonet** [comment]
:) I’m happy I found this space issue... 😜

**Pull request merged**